content/en/docs/Concepts/operators-on-cluster.md

```diff
@@ -0,0 +1,21 @@
+# When to use operators on a cluster
+
+
+Operators are good for automating the knowledge of how to operate a complex system. Operators are a high-privilege component and by design, they run persistently inside your cluster. An operators can handle features like automatic scaling in response to load, backup and restore, integration with metric systems like Prometheus, and even failure detection and automatic tuning in response to usage patterns.
+
+# How does OLM help you use operators on a cluster?
+
+OLM requires you to provide metadata about your operator in order to ensure that it can be kept running safely on a cluster, and to provide information about how updates should be applied as you publish new versions of your operator. A ClusterServiceVersion (CSV) represents a particular version a running operator on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc. It declares `owned`/`required` CRDs, cluster requirements, and install strategy that tells OLM how to create required resources and set up the operator as a [deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/).
+
+As a cluster administrator, you can install an Operator from the OperatorHub using the OpenShift Container Platform web console or the CLI. You can then subscribe the Operator to one or more namespaces to make it available for developers on your cluster.
+
+During installation, you must determine the following initial settings for the Operator:
+
+Installation Mode
+Choose All namespaces on the cluster (default) to have the Operator installed on all namespaces or choose individual namespaces, if available, to only install the Operator on selected namespaces.
```

Comment:
For installation mode, what should I know? What are the available options? and what does that translate to? For example, All Namespaces means what? It means that the Operator can WATCH for resources in ALL namespaces. 
And make sure you reference them the same way as other places. If it is AllNamespaces in other places then use that term. 

And the topic "you must determine the following initial settings" means that I as the reader of this want to know what I will need. So if this is explicitly what defines what I need to specify then we might have to mention OperatorGroups and how these installation modes affect it. For example, AllNamespaces mode means that we do not set the TargetNamespace on the OperatorGroup.

You can see how the others affect the OperatorGroup targetNamespaces from this code: https://github.com/operator-framework/operator-sdk/blob/master/internal/olm/operator/registry/operator_installer.go#L367-L380